skimage/data/_binary_blobs.py

```diff
@@ -18,9 +18,12 @@ def binary_blobs(length=512, blob_size_fraction=0.1, n_dim=2,
     volume_fraction : float, default 0.5
         Fraction of image pixels covered by the blobs (where the output is 1).
         Should be in [0, 1].
-    seed : int, optional
-        Seed to initialize the random number generator.
-        If `None`, a random seed from the operating system is used.
+    seed : {None, int, `numpy.random.Generator`}, optional
```

Comment:
This is a new feature and should be highlighted. I Think we things like `.. versionadded::`

Reply:
Indeed, also not clear what to choose as per #5359. And probably I should add it as keyword argument only with `..., *, seed`. What should I put for the version?

Reply:
Yes, please make any added parameters keyword-only. 

Use `0.19` for the versionadded statement.

Reply:
ok thanks @grlee77. And for the naming?

Reply:
I don't think we reached any conclusion in #5359 on the preferred name, but I am personally leaning toward `random_state`. Unless there are objections, we could use that for any newly added parameter here. I wouldn't rename any of the pre-existing parameters in this PR. Let's leave that to a potential follow-up.

Reply:
To be clear, in this case I did not add the parameter but just the documentation and called behind `default_rng ` instead of `RandomState`. But there are other places where I added the parameter. Correct me if I am wrong, but I should just add versionadded info to the new places but not here.